experimental/host/beagle/black/black.go

```diff
@@ -97,10 +98,15 @@ var (
 // Beagleboard present.
 func Present() bool {
 	if isArm {
-		// TODO: check for beaglebone black in particular
-		_, err := os.Stat("/sys/firmware/devicetree/base/bone_capemgr")
+		buf, err := ioutil.ReadFile("/sys/firmware/devicetree/base/model")
```

Comment:
Oh there's already a function for this; I prefer the function to be reused:
https://godoc.org/periph.io/x/periph/host/distro#DTModel
Sorry I hadn't been clear about this.

If /proc/device-tree/model doesn't work well, update https://github.com/google/periph/blob/master/host/distro/devtree.go#L47 and https://github.com/google/periph/blob/master/host/distro/devtree.go#L57 instead.